source/patterns/@aws-solutions-constructs/aws-openapigateway-lambda/lib/index.ts

```diff
@@ -77,6 +36,13 @@ export interface OpenApiGatewayToLambdaProps {
    * Local file asset of the OpenAPI spec file.
    */
   readonly apiDefinitionAsset?: Asset;
+  /**
+   * The OpenApi spec represented as a json object to be included in the CloudFormation template.
```

Comment:
The `AS A JSON OBJECT` bit of this is important, as a yaml-defined openapi spec wouldn't work, though is probably the more common format to define it. How can we better call that out in this comment to make sure customers know?

Reply:
To me the key point is that this definition will be embedded in the template, so the comment is focused on that. The README is slightly different, do you like that any better?

Reply:
Not passing JSON would cause this to die at synth time, right? I think its fine.